### PR TITLE
JavaScript Capstone Project

### PR DESCRIPTION
**Pull request by StarSheriff2**
Hi, Code Reviewer 😄:

In this pull request, we built all the main functionality of the website according to the requirements of the client.

The page consumes data stored in 2 external APIs. We pull all the main content of the page from one and store all interactions on the page, such as likes and comments, from the other one.

We used async and await to handle all API calls. We compiled the project using Webpack.

The Kanban methodology was employed to work in this project, where each developer owned one main feature implementation.

The page consists of a homepage that displays one category of elements retrieved from the API.

Features:

- Browse through a list of all elements in the monsters category
- Like any monster
- See information about each monster by clicking on the comment button
- Comment on any monster card

In this link, you will find our video presentation: [https://www.loom.com/share/d6ef004d5be24461957ab0480a337a3b](https://www.loom.com/share/d6ef004d5be24461957ab0480a337a3b)

### CODE REVIEW

File: src/app/DOM/pop_up_window.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    item.addEventListener('click', () => {
+      body.removeChild(item.parentElement);
+    });
+  }
+
+  static async removeModal() {
+    const body = document.querySelector('body');
+    body.addEventListener('click', (e) => {
+      const modal = document.querySelector('.modal-container');
+      if (!modal.contains(e.target)) {
```
Comment (Davidosky007) [inline src/app/DOM/pop_up_window.js]: * This line throws an error whenever i click the comments and like buttons kindly fix it 🙏 
image

File: src/app/DOM/pop_up_window.js [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+      <div class="comments-info">
+    </section>
+    <section class="form-container">
+      <h2 class="add-title">Add a comment</h2>
+      <form class="comments-form" action="submit">
+        <input id="user" type="text" placeholder="Your Name">
+        <textarea name="comments" id="comments-text" cols="30" rows="10" placeholder="Your Insights"></textarea>
+      </form>
+      <div class="btn-container">
+        <button class="comment-btn" type="button">Add comment</button>
```
Comment (Davidosky007) [inline src/app/DOM/pop_up_window.js:53]: [OPTIONAL]
* This is an optional suggestion as it is not originally a requirement for this project 
* I think clearing all input fields after adding comments would improve the user experience or don't you think so?

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
# Hi @StarSheriff2 @jchernandez87 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
* Nice job 💪 so far 
* Video presentation was perfect it meets all requirements 
* We just have one little bug 🐛 to fix 🧑‍🔧 
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- I suggest that API keys be secretly used and not exposed publicly as they can be used by anyone with access to your code which we don't want to happen. [Read more on best practices with API keys ](https://www.freecodecamp.org/news/private-api-keys/)
- Check the comments below for an optional suggestion

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by abruzy — APPROVED** [review]
Hi Team 👋🏽, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚢  
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**